File: learn-kotlin/src/main/kotlin/com/greatdreams/learn/kotlin/coroutine/mainprogram.kt
package com.greatdreams.learn.kotlin.coroutine

import com.greatdreams.learn.kotlin.with.WithTest
import kotlinx.coroutines.experimental.*

object MainProgram {
    @JvmStatic fun main(args: Array<String>) {
        WithTest.main(args)

        launch {
            delay(2000L)
            println("coroutines!")
        }
        println("Hello, ")
        runBlocking {
            delay(3000L)
        }

    }
}

File: learn-scalanlp/src/main/scala/com/greatdreams/learn/scalanlp/MainProgram.scala
package com.greatdreams.learn.scalanlp
import breeze.linalg.DenseVector

object MainProgram {
  def main(args: Array[String]): Unit = {
    println("Hello, scalanlp!")
    val x = DenseVector.zeros[Double](5)
    println(x)
  }
}
